python/ray/experimental/placement_group.py

```diff
@@ -1,15 +1,70 @@
+import random
 from typing import (List, Dict)
 
 import ray
+from ray._raylet import PlacementGroupID
 
 
 class PlacementGroup:
     """A handle to a placement group.
+
+    Args:
+        id: Placement group id.
+        bundles: List of bundles.
     """
 
-    def __init__(self, id, bundle_count):
+    @staticmethod
+    def empty():
+        return PlacementGroup(PlacementGroupID.nil(), [])
+
+    def __init__(self, id: PlacementGroupID, bundles: List[Dict[str, float]]):
         self.id = id
-        self.bundle_count = bundle_count
+        self.bundles = bundles
+
+    def ready(self):
+        """Returns an object ID to check ready status."""
+        worker = ray.worker.global_worker
+        worker.check_connected()
+
+        @ray.remote(num_cpus=0, max_calls=0)
+        def bundle_reservation_check(placement_group):
+            return placement_group
+
+        assert len(self.bundles) != 0, (
+            "ready() cannot be called on placement group object with a "
+            f"bundle length == 0, current bundle length: {len(self.bundles)}")
+
+        bundle_index = random.randint(0, len(self.bundles) - 1)
```

Comment:
Why it returns a random index?

Reply:
There are 2 solutions here. 

1. Schedule tasks for all bundles.
2. Schedule one task to one random bundle.

I implemented 2 because once we guarantee the atomic creation, 2 can guarantee to wait until placement group is actually created. I can add a comment if it is confusing.

Reply:
Added a comment.

Reply:
If the placement group scheduling is successful, but node is offline immediately. And we choose a placed bundle, is that going to be a problem?

Reply:
I see. In this case, 

1. If the task is already scheduled => This will raise exception, which will be the same behavior although we schedule bundles in all nodes.
2. If the task is not scheduled yet => It will wait until the bundle is created. I think this will be also the same when we schedule tasks on all bundles (which I believe is expected behavior as well). 

So, I think it should be fine? I'd like to hear your thoughts tho!

Reply:
Instead of choosing a random index, can we always choose bundle 0?

Reply:
Sure. I don't think there will be any side effect from that change.